modules/hal_nordic/nrfx/CMakeLists.txt

```diff
@@ -42,6 +42,11 @@ zephyr_compile_definitions_ifdef(CONFIG_NRF_SECURE_APPROTECT_USER_HANDLING
 zephyr_library_compile_definitions_ifdef(CONFIG_NRF_TRACE_PORT
                                  ENABLE_TRACE)
 
+zephyr_compile_definitions_ifdef(CONFIG_SOC_NRF5340_CPUAPP
```

Comment:
Nothing wrong on this line, but the commit message 95616a5d029f0d3ab284bf41ab5719c82e2dfcbf says:
> memory region is ecluded from use.

I assume you mean: `excluded`